README.md

```diff
@@ -174,15 +174,16 @@ database does not have the private key:
 
 ### SecretStack.use(require('ssb-db')) => sbot
 
-Adds ssb-db persistence to a secret-stack setup.
+Adds `ssb-db` persistence to a [secret-stack](https://github.com/ssbc/secret-stack) setup.
 Without other plugins, this instance will not have replication
-or querying. Loading ssb-db directly is useful for testing,
+or querying. Loading `ssb-db` directly is useful for testing,
 but it's recommended to instead start from a plugin bundle like [ssb-server](https://github.com/ssbc/ssb-server)
 
-Because of legacy reasons, all the ssb-db methods are mounted on the top level object,
-so it's `sbot.get` instead of `sbot.db.get` as it would be with all the other `ssb-*` plugins.
+> Because of legacy reasons, all the `ssb-db` methods are mounted on the top level object,
+so it's `ssbServerWithDb.get` instead of `ssbServerWithDb.db.get` as it would be with all the other `ssb-*` plugins.
+> In the API docs below, we'll just call it `db`
```

Comment:
@christianbundy I added an extra note about that here, but I'll change the `ssbServerWithDb` reference

Reply:
Oh nice! Totally slipped my brain.